libraries/AC_AttitudeControl/AC_WeatherVane.cpp

```diff
@@ -24,7 +24,7 @@ const AP_Param::GroupInfo AC_WeatherVane::var_info[] = {
     // @Param: ENABLE
     // @DisplayName: Enable
     // @Description: Enable weather vaning.  When active, the aircraft will automatically yaw into wind when in a VTOL position controlled mode. Pilot yaw commands overide the weathervaning action.
-    // @Values: 0:Disabled,1:Nose into wind,2:Nose or tail into wind,3:Side into wind,4:tail into wind
+    // @Values: -1: Only use during takeoffs or landing, see _TAKEOFF and/or _LAND parameters, 0:Disabled,1:Nose into wind,2:Nose or tail into wind,3:Side into wind,4:tail into wind
```

Comment:
does not match other param patterns in this library

Reply:
I don't understand this. How can you specify: Nose into the wind + Only use on takeoff and landing?

Reply:
if you are pointing into the wind and takeoff, with into the wind set, it will weathervane into the wind

Reply:
Sorry to be obtuse @Hwurzburg , but if the value is set to -1 (for takeoff and landing) you can't set it to "into the wind"  = 1 - right? How can it be -1 and 1 at the same time?

Reply:
-1 makes only the takeoff and land override params set the direction and only in landing or takeoff mode, not flight mode, as it is now...especially problematic in copter...qplane only uses in qloiter flight mode, but copter uses it in guided and auto

Reply:
Ok maybe the docs will make it clear. Right now I just don't understand how I would set this up.

Reply:
you can have wvane: while flying in guided/auto in copter or qloiter in qplane, or takeoffs only (NAV takeoffs), or landings only(QLAND, LAND, Nav Lands), or takeoffs and landings, or all three

currently you ALWAYS have wvane in guided or auto while flying if you enable it ...cant get just for landings, for example (the most commonly need in Qplane)

Reply:
I get that - what I don't get is how you set the _direction_ once you have chosen -1 for takeoffs/landings.

This field says it sets the direction (if set to 1, 2, 3 or 4) - is there another way to set the direction?

Because if you set it to -1, how can you set the direction?

Reply:
read the current weathervaning wiki:https://ardupilot.org/plane/docs/quadplane-weathervaning.html?